dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/JdbcEventAnalyticsManager.java

```diff
@@ -605,6 +605,19 @@ private String getOrgDescendantsSqlSnippet( String orgUnitAlias,
             .collect( joining( " and " ) );
     }
 
+    private String getSqlSnippetForFallbackCoordinateFields( EventQueryParams params )
```

Comment:
I think would be cool to document this method by showing an example of an output of the "snippet".